### DISCUSSION TITLE
Auto Center on /loc for map

### DISCUSSION THREAD

**Discussion by patrickisfrench** [smasherprog/EqTool#27] [Ideas]
Auto Center on /loc for map, is this planned later?

**Comment by smasherprog** [comment]
yeah its on my list https://github.com/smasherprog/EqTool#to-do-list-eqtool
Auto follow map. . . .  Lots of things to do!!

**Reply by patrickisfrench** [reply]
oh nice, this is great. the app doesn't scale very well on a 4k monitor either. is this an easy solve?

**Comment by smasherprog** [comment]
You can adjust the font size in the settings window, does that help?

**Reply by patrickisfrench** [reply]
I have tried font adjustments but if i move the app over to my 1440p monitor everything is sharper. apps that support windows scaling look a bit cleaner on the 4K, otherwise there's like a blurry haze over the clarity of everything. subtle, but noticeable.

**Comment by smasherprog** [comment]
I am still working on this, other things have continued to get ahead of it, but its still in progress.
